tests/bofire/data_models/specs/strategies.py

```diff
@@ -125,6 +125,8 @@
     lambda: {
         "domain": domain.valid().obj().dict(),
         "formula": "linear",
+        "optimization_strategy": "default",
+        "verbose": False,
```

Comment:
What is this ´verbose´ flag doing?

Reply:
If true, the method prints out information about the optimization process. So for BaB, the number of branches explored and the current lower bound.